release-notes/opensearch-dashboards.release-notes-1.0.0-rc1.md

```diff
@@ -0,0 +1,104 @@
+## Version 1.0.0-rc1 Release Notes
+
+* __Fix beta doc links, truncated help menu label (#382)__
+
+    [Tommy Markley](mailto:markleyt@amazon.com) - Wed, 26 May 2021 15:54:42 -0700
```

Comment:
I think we should consider the format for these release notes for GA. It looks like the engine team mentioned that as well. I think would be nice to have links to the PRs at least.

Reply:
Sure. We can sync up with OpenSearch team to formalize the format.